licneses/libpwquality

```diff
@@ -0,0 +1,16 @@
+Redistribution and use in source and binary forms of libpwquality, with
```

Comment:
Please correct the filename.
You are calling the path: `licenses` but you are calling the directory `licneses`

Reply:
@aliceinwire thanks. I made new PR to warrior branch. https://github.com/miraclelinux/meta-debian-extended/pull/350